x-pack/plugins/stack_alerts/server/rule_types/es_query/rule_type.ts

```diff
@@ -152,7 +154,15 @@ export function getRuleType(
     actionGroups: [{ id: ActionGroupId, name: actionGroupName }],
     defaultActionGroupId: ActionGroupId,
     validate: {
-      params: EsQueryRuleParamsSchema,
+      params: {
+        validate: (object: unknown) => {
+          const validated = EsQueryRuleParamsSchema.validate(object);
```

Comment:
are we validating max size in non-serverless? Looks like we are in the UI but I don't see a check for that inside `EsQueryRuleParamsSchema`?

Reply:
Yes, the schema definition for size has the max size set so it should fail if you go above that number.